README.md

```diff
@@ -81,7 +81,7 @@ previous section, for example:
 
     curl -fLO https://github.com/erlef/otp_builds/releases/download/OTP-27.1.2/otp-aarch64-apple-darwin.tar.gz
     checksum=$(curl -fsSL https://github.com/erlef/otp_builds/raw/main/builds/aarch64-apple-darwin.csv | grep OTP-27.1.2, | cut -d"," -f4)
-    sha256sum otp-aarch64-apple-darwin.tar.gz --check $checksum
+    echo "$checksum otp-aarch64-apple-darwin.tar.gz" | sha256sum --check
```

Comment:
was the previous wrong, not portable, or is it a matter of preference?

Reply:
This worked for me, could you check?

```
echo 'Hello, world!' > otp-aarch64-apple-darwin.tar.gz
checksum=d9014c4624844aa5bac314773d6b689ad467fa4e1d1a50a1b8a99d5a95f72ff5
sha256 otp-aarch64-apple-darwin.tar.gz --check "$checksum"
```

i.e. sha256 instead of sha256sum

Reply:
Just checked, sha256sum is installed from brew - completely missed that 🤦

```
which sha256sum
/opt/homebrew/bin/sha256sum
```

I am on older macOS, that must explain why I don't have `sha256`:

```
ProductName:            macOS
ProductVersion:         14.1
BuildVersion:           23B2073
```